src/vtp/show_contest.py

```diff
@@ -25,17 +25,17 @@
 
 """
 
+import argparse
+import logging
 # pylint: disable=wrong-import-position
 import os
 import re
 import sys
-import argparse
-import logging
 from logging import error
 
 # Local imports
-from common import Globals, Shellout
-from election_config import ElectionConfig
+from utils.common import Globals, Shellout
+from utils.election_config import ElectionConfig
```

Comment:
Ditto: use relative imports.

```suggestion
from .utils.common import Globals, Shellout
from .utils.election_config import ElectionConfig
```